src/Server.ts

```diff
@@ -1,5 +1,7 @@
 import * as express from 'express';
+import * as bodyParser from 'body-parser';
 import Iconfig from './config/iconfig';
+import { errorHandler, notFoundRoute } from './libs/routes/index';
 import { eventNames } from 'cluster';
```

Comment:
why is this used? Please make sure to remove unnecessary importing before pushing

Reply:
Done